package.json

```diff
@@ -95,6 +96,7 @@
     "clipboard": "1.5.5",
     "commander": "2.8.1",
     "css-loader": "0.17.0",
+    "csv": "0.4.6",
```

Comment:
Good call, I was using the transform portion of the package at one point as well but that went away.